### PR TITLE
Update log-tailing.md

### PR DESCRIPTION
**Pull request by schmurfy**
**Description**

Fixes a typo.

### CODE REVIEW

**Comment by schmurfy** [comment]
@googlebot I signed it!

File: docs/content/en/docs/pipeline-stages/log-tailing.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Log Tailing is **disabled by default** for `run` mode; it can be enabled with th
 
 
 ## Log Structure
-To view log structure, run `skaffold run --tail` on [examples microserices](https://github.com/GoogleContainerTools/skaffold/tree/master/examples/microservices)
+To view log structure, run `skaffold run --tail` on [examples microservices](https://github.com/GoogleContainerTools/skaffold/tree/master/examples/microservices)
```
Comment (blaggacao) [inline docs/content/en/docs/pipeline-stages/log-tailing.md:18]:
```suggestion
To view log structure, run `skaffold run --tail` on [examples microservices](https://github.com/GoogleContainerTools/skaffold/tree/master/examples/microservices)
```
```suggestion
To view log structure, run `skaffold run --tail` on [example microservices](https://github.com/GoogleContainerTools/skaffold/tree/master/examples/microservices)
```

**Comment by briandealwis** [comment]
@blaggacao it doesn't appear that you've signed the CLA?  Could you please visit https://cla.developers.google.com/ to sign?

**Comment by briandealwis** [comment]
The text was referencing the `examples/microservices` path with Skaffold, so I restored @schmurfy's fix and then fixed that up.

**Pull request merged**